1-mandatory/2-quotegenerator/quotes.js

```diff
@@ -490,3 +513,18 @@ const quotes = [
     author: "Zig Ziglar",
   },
 ];
+
+let quotePara = document.getElementById("quote");
+let authorName = document.getElementById("author");
+let newQuoteButton = document.getElementById("quoteGenerator");
+
+
+function setQuote() {
+  let selectedQuote = pickFromArray(quotes);
+  quotePara.innerText = selectedQuote.quote;
+  authorName.innerText = `- ${selectedQuote.author}`;
+}
+
+newQuoteButton.addEventListener("click", setQuote);
+
+setQuote();
```

Comment:
This does what its supposed to, by I find the code a bit hard to read with all the extra commented bits. Try and get into a habit of having a tidy-up of your code before submitting, I believe there's a style guide that explains what to look out for!